grass7/raster/r.stream.basins/io.c

```diff
@@ -75,8 +75,8 @@ int ram_read_map(MAP * map, char *input_map_name, int check_res,
     G_get_window(&this_window);
     Rast_get_cellhd(input_map_name, mapset, &cellhd);
     if (check_res)
-	if (this_window.ew_res != cellhd.ew_res ||
-	    this_window.ns_res != cellhd.ns_res)
+	if (fabs(this_window.ew_res - cellhd.ew_res) >= 1e-10 ||
+	    fabs(this_window.ns_res - cellhd.ns_res) >= 1e-10)
```

Comment:
Use GRASS_EPSILON.

Reply:
`GRASS_EPSILON` is `1e-15` and the difference was greater than that (`-2.2311e-12` EW and `9.45022e-13` NS) *yesterday*. Now, I restarted in the same mapset for testing, the error is `0`. Interesting.. there were no changes in the mapset. All I did is exited and restarted the same mapset. In short, `GRASS_EPSILON` would not have worked yesterday, but now it works because of an unknown reason. Let me try it from scratch.